lib/Sema/SemaExpr.cpp

```diff
@@ -12465,23 +12466,29 @@ ExprResult Sema::CreatePositionalParameterExpr(unsigned Index, QualType QT) {
   return new (Context) PositionalParameterExpr(Index, QT);
 }
 
-ExprResult Sema::ActOnBoundsCastExpr(Scope *S, SourceLocation LParenLoc,
-                                     ParsedType D,
-                                     RelativeBoundsClause *RelativeClause,
-                                     SourceLocation RParenLoc, Expr *E1,
-                                     Expr *E2, Expr *E3,
-                                     BoundsCastExpr::Kind kind) {
+ExprResult Sema::ActOnBoundsCastExpr(
+    Scope *S, SourceLocation OpLoc, tok::TokenKind Kind,
+    SourceLocation LAngleBracketLoc, ParsedType D,
+    SourceLocation RAngleBracketLoc, RelativeBoundsClause *RelativeClause,
+    SourceLocation LParenLoc, SourceLocation RParenLoc, Expr *E1, Expr *E2,
+    Expr *E3, BoundsCastExpr::SyntaxType SyntaxType) {
   RangeBoundsExpr *Range = nullptr;
   TypeSourceInfo *castTInfo;
-  GetTypeFromParser(D, &castTInfo);
+  QualType DestTy = GetTypeFromParser(D, &castTInfo);
```

Comment:
yes. I should remove this code. I was needed previously but it's not now. Thanks.